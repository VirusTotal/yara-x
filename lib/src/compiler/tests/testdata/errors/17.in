rule test {
  strings:
    $a = "foo"
  condition: 
    @a["1"] == 0x100
}