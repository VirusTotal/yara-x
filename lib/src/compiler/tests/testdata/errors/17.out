error: wrong type
 --> line:5:8
  |
5 |     @a["1"] == 0x100
  |        ^^^ expression should be `integer`, but is `string`
  |